### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by parminpatel28**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,3 +35,4 @@ yarn-error.log*
 .vercel
 package-lock.json
 .vscode
+.env.sample
```
Comment (shivam-sh) [inline .gitignore:38]: ?

**Comment by shivam-sh** [comment]
There’s a bunch of graph stuff in the changed files, can the cards work be isolated from that before we merge it?

**Comment by parminpatel28** [comment]
> There’s a bunch of graph stuff in the changed files, can the cards work be isolated from that before we merge it?

It's a result of a bad merge

**Comment by shivam-sh** [comment]
> > There’s a bunch of graph stuff in the changed files, can the cards work be isolated from that before we merge it?
> 
> It's a result of a bad merge

Yeah, can we get that fixed?

**Pull request closed without merging**